packages/editor/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@theia/core": "^0.7.0",
     "@theia/languages": "^0.7.0",
+    "@theia/workspace": "^0.7.0",
```

Comment:
I assume this is a mistake?
https://github.com/theia-ide/theia/pull/5478#discussion_r295243996 previously said to revert it.

Reply:
Oops